keyboards/gergo/keymaps/mim/rules.mk

```diff
@@ -0,0 +1,45 @@
+#----------------------------------------------------------------------------
+# make gergo:germ:dfu
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+# Firmware options
+#BALLER = no 			# Enable to ball out
+#BALLSTEP = 20  			# Multiple in px to move, multiplied by layer number
+#SCROLLSTEP = 1 			# Lines to scroll with ball
+#MOUSEKEY_ENABLE = yes  		# Mouse keys, needed for baller
+
+# Combo options
+COMBO_ENABLE = yes
+#Tap dance options
+TAP_DANCE_ENABLE = yes
+# Dynamic macros
+DYNAMIC_MACRO_ENABLE = yes
+
+
+#Debug options
+VERBOSE 		 = yes
+DEBUG_MATRIX_SCAN_RATE   = no
+DEBUG_BALLER 		 = no
+DEBUG_MATRIX		 = yes
+
+# A bunch of stuff that you shouldn't touch unless you
+# know what you're doing.
+#
+# No touchy, capiche?
+SRC += matrix.c i2c_master.c
```

Comment:
This should be removed, as it exists at the kb level, so no need to re-include it here. 
```suggestion
```